charts/airbyte-bootloader/README.md

```diff
@@ -16,11 +16,11 @@ Helm chart to deploy airbyte-bootloader
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
 | enabled | bool | `true` |  |
-| global.database.secretName | string | `"test-postgresql"` |  |
```

Comment:
I'm not opposed exactly, but why are we removing the placeholder strings?

Reply:
So it won't be used in actual deployment. We're determining configmap and secret names dynamically using Release name.
If this value is supplied it will use it instead of the dynamic mapping